options.go

```diff
@@ -307,6 +307,19 @@ func EnableNATService() Option {
 	}
 }
 
+// OverrideNATServiceThrottling changes the default rate limiting configured in helping
+// other peers determine their reachability status. When set, the host will limit
+// the number of requests it responds to in each 60 second period to the set
+// numbers. A value of '0' disables throttling.
+func OverrideNATServiceThrottling(globalThrottle, perPeerThrottle int) Option {
+	return func(cfg *Config) error {
+		cfg.AutoNATThrottling = true
```

Comment:
Should default to _on_, no?

Reply:
throttling is on by default - but is over over-ridden when this option is set. Updating to a struct should make the libp2p-level config less confusing.